aws/cloudflare-lb-pools/providers.tf

```diff
@@ -0,0 +1,8 @@
+terraform {
+  required_providers {
+    cloudflare = {
+      source  = "cloudflare/cloudflare"
+      version = "~> 3.0"
```

Comment:
@Mshahidtaj shouldn't this be variable?

Reply:
> cannot be a variable in the required providers if I remember correctly. Right?

Thats right @spirosoik , @stafot unfortunately, the provider does not support variables.

Reply:
Thanks